src/bin/vip-import-media.js

```diff
@@ -94,9 +94,10 @@ Are you sure you want to import the contents of the url?
 } )
 	.command( 'status', 'Check the status of the current running import' )
 	.option( 'exportFileErrorsToJson', 'Export any file errors encountered to a JSON file instead of a plain text file', false )
+	.option( 'overwriteExisting', 'Overwrite any existing files', false )
```

Comment:
I'd recommend keeping Parker and CLI in sync. So `overwriteExistingFiles`

Reply:
Addressed here: https://github.com/Automattic/vip/pull/739/commits/f0a4738c5c1dd4989a827dae951d111394e84fec